src/modules/template-details/TemplateDetails.tsx

```diff
@@ -10,115 +10,114 @@ import ShareIcon from '@material-ui/icons/Share';
 
 import { Button } from 'ui';
 
+import { getTemplateDetails, Template } from 'core/api';
+
 import csx from './TemplateDetails.scss';
 
+
+
 interface TemplateDetailsProps extends RouteChildrenProps<{ id: string }> {}
 
+// TODO - REPLACE MISSING PROPERTIES WHEN ENDPOINT WILL BE FINISHED
+// TODO - CONNECT EDIT
+
 const TemplateDetails = ({ match }: TemplateDetailsProps) => {
-  const MOCKED_TECH = ['PWA', 'React', 'JavaScript', 'TypeScript', 'MVP'];
-  const MOCKED_PATTERNS = ['PWA', 'MVP'];
   const MOCKED_TECH_STACK = ['React', 'Angular', 'Vue'];
   const MOCKED_AUTHORS = [
     'https://dummyimage.com/64x64/000/fff.png',
     'https://dummyimage.com/64x64/000/fff.png',
     'https://dummyimage.com/64x64/000/fff.png'
   ];
 
+  const [template, setTemplate] = useState<Template>(null);
+
   const mapList = (list: string[]) => list.map((item) => <li key={item}>{item}</li>);
```

Comment:
This map functions should be components - or should be moved from component render method - you creating instance of this function in every render - same for mapImages. Creating component for this case is too much but you can move this functions from component definition